### ISSUE TITLE
Add licenses everywhere

### ISSUE THREAD

**Issue by cugu** [kaitai-io/kaitai_struct#474]
Missing license:
- [x] https://github.com/kaitai-io/kaitai_struct_benchmarks (https://github.com/kaitai-io/kaitai_struct_benchmarks/pull/2)
- [x] https://github.com/kaitai-io/kaitai_struct_tests (https://github.com/kaitai-io/kaitai_struct_tests/pull/57)
- [x] https://github.com/kaitai-io/kaitai_ci_ui (https://github.com/kaitai-io/kaitai_ci_ui/pull/2)
- [x] https://github.com/kaitai-io/kaitai_struct_doc (https://github.com/kaitai-io/kaitai_struct_doc/pull/30)
- [x] https://github.com/kaitai-io/kaitai_struct_gui
- [x] https://github.com/kaitai-io/ci_targets (https://github.com/kaitai-io/ci_targets/pull/4)
- [x] https://github.com/kaitai-io/kaitai_struct_nim_runtime (https://github.com/kaitai-io/kaitai_struct_nim_runtime/issues/1)
- [x] https://github.com/kaitai-io/java_bytecode.ksy (https://github.com/kaitai-io/java_bytecode.ksy/pull/1)

No `LICENSE`file:
- [x] https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime (https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/pull/32)
- [x] https://github.com/kaitai-io/kaitai_struct_java_runtime (https://github.com/kaitai-io/kaitai_struct_java_runtime/pull/19)
- [x] https://github.com/kaitai-io/kaitai_struct_perl_runtime (https://github.com/kaitai-io/kaitai_struct_perl_runtime/pull/4)
- [x] https://github.com/kaitai-io/kaitai_struct_python_runtime (https://github.com/kaitai-io/kaitai_struct_python_runtime/pull/34)
- [x] https://github.com/kaitai-io/kaitai_struct_ruby_runtime (https://github.com/kaitai-io/kaitai_struct_ruby_runtime/pull/3)
- [x] https://github.com/kaitai-io/kaitai_struct_swift_runtime (https://github.com/kaitai-io/kaitai_struct_swift_runtime/pull/4)
- [ ] https://github.com/kaitai-io/kaitai_struct_typescript_runtime (https://github.com/kaitai-io/kaitai_struct_typescript_runtime/pull/1)
- [x] https://github.com/kaitai-io/dicom.ksy (https://github.com/kaitai-io/dicom.ksy/pull/1)
- [x] https://github.com/kaitai-io/edid.ksy (https://github.com/kaitai-io/edid.ksy/pull/1)
- [x] https://github.com/kaitai-io/windows_resource_file.ksy (https://github.com/kaitai-io/windows_resource_file.ksy/pull/2)

Unsure:
- [ ] https://github.com/kaitai-io/kaitai_struct (does not really contain any files)
- [ ] https://github.com/kaitai-io/kaitai-io.github.io (GFDL? GPL?)
- [ ] https://github.com/kaitai-io/ci_artifacts (does a license for artifacts make sense?)
- [ ] https://github.com/kaitai-io/kaitai_struct_webide_vue (stale repo)
- [ ] https://github.com/kaitai-io/kaitai_struct_examples (stale repo)

Not readable by GitHub:
- [ ] https://github.com/kaitai-io/kaitai_struct_webide

**Comment by cherue** [comment]
This can be updated with check marks for:
- kaitai_struct_tests
- kaitai_ci_ui

Now that the tests are licensed kaitai-io/ci_targets#4 can be merged and checked off, too.

**Comment by GreyCat** [comment]
I would actually try to do it another way: let's strip `ci_targets` from everything but generated code + minimal possible set of CI instructions, and clearly mark it as technical repo, derivative of the compiler + tests + runtimes?

**Comment by cherue** [comment]
SGTM. So that would mean moving all the `prepare-*` scripts to the test repo and using the `config` from the test repo too, right?

**Comment by GreyCat** [comment]
Yep. Ideally, I'd even want to switch to into the mode when we won't depend on any particular proprietary CI's init/prepare instructions and just use our own, say, Docker containers, so exact CI test runs can be easily reproducible on local machine.

**Comment by GreyCat** [comment]
Extracted this as a separate idea: https://github.com/kaitai-io/kaitai_struct/issues/714
